src/components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+import { ComponentPropsWithoutRef } from 'react';
+
+export interface RadioButtonProps extends ComponentPropsWithoutRef<'div'> {
+  radioId: string;
```

Comment:
You can use .`..rest` for all native attrs: id, name etc... No need an specific one.